lib/vmdb/settings/validator.rb

```diff
@@ -137,6 +154,25 @@ def server(data)
           end
         end
 
+        if keys.include?(:rate_limiting)
+          %i[api_login request ui_login].each do |limiting_section|
+            next if data.rate_limiting[limiting_section].blank?
+
+            limit  = data.rate_limiting[limiting_section][:limit]
+            period = data.rate_limiting[limiting_section][:period]
+
+            unless is_integer?(limit) || limit.blank?
```

Comment:
is_integer? will handle blanks as well, so the blank check is not needed

```ruby
is_integer?(nil) # => false
is_integer?("")  # => false
```

Reply:
Same thing as above.  Fails specs:

```console
﻿$ bundle exec rspec spec/lib/vmdb/settings/validator_spec.rb
** Using session_store: ActionDispatch::Session::MemoryStore
** ManageIQ master, codename: Morphy

Randomized with seed 19849
..F........F..............................F.....................

Failures:

  1) Vmdb::Settings::Validator {:server=>{:rate_limiting=>{:api_login=>{:limit=>nil, :period=>nil}}}}
     Failure/Error: expect(validator.valid?).to be expected

       expected true
            got false
     # ./spec/lib/vmdb/settings/validator_spec.rb:93:in `block (3 levels) in <top (required)>'
     # /Users/nicklamuro/.gem/ruby/2.6.6/gems/webmock-3.13.0/lib/webmock/rspec.rb:37:in `block (2 levels) in <top (required)>'

  2) Vmdb::Settings::Validator {:server=>{:rate_limiting=>{:request=>{:limit=>nil, :period=>nil}}}}
     Failure/Error: expect(validator.valid?).to be expected

       expected true
            got false
     # ./spec/lib/vmdb/settings/validator_spec.rb:93:in `block (3 levels) in <top (required)>'
     # /Users/nicklamuro/.gem/ruby/2.6.6/gems/webmock-3.13.0/lib/webmock/rspec.rb:37:in `block (2 levels) in <top (required)>'

  3) Vmdb::Settings::Validator {:server=>{:rate_limiting=>{:ui_login=>{:limit=>nil, :period=>nil}}}}
     Failure/Error: expect(validator.valid?).to be expected

       expected true
            got false
     # ./spec/lib/vmdb/settings/validator_spec.rb:93:in `block (3 levels) in <top (required)>'
     # /Users/nicklamuro/.gem/ruby/2.6.6/gems/webmock-3.13.0/lib/webmock/rspec.rb:37:in `block (2 levels) in <top (required)>'

Finished in 2.72 seconds (files took 10.66 seconds to load)
64 examples, 3 failures

Failed examples:

rspec ./spec/lib/vmdb/settings/validator_spec.rb[1:34] # Vmdb::Settings::Validator {:server=>{:rate_limiting=>{:api_login=>{:limit=>nil, :period=>nil}}}}
rspec ./spec/lib/vmdb/settings/validator_spec.rb[1:39] # Vmdb::Settings::Validator {:server=>{:rate_limiting=>{:request=>{:limit=>nil, :period=>nil}}}}
rspec ./spec/lib/vmdb/settings/validator_spec.rb[1:44] # Vmdb::Settings::Validator {:server=>{:rate_limiting=>{:ui_login=>{:limit=>nil, :period=>nil}}}}
```

Reply:
Correction: I had bad test cases as I don't think we should accept `nil` as a valid value.  Was admittedly copying from the `data.listening_port` validations above, so that was my bad.

Changing the specs and making the change.

Reply:
Fixed.